src/main/java/mate/academy/internetshop/controller/AddItemToBucketController.java

```diff
@@ -22,10 +22,12 @@
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        String itemId = req.getParameter("item_id");
+        Long userId = (Long) req.getSession(true).getAttribute("userId");
```

Comment:
за що відповідає параметр true?

Reply:
getSession() and getSession(true) make the same actions(if exist session, then return it. Else create new), I know and understand it. 
I chose getSession(true), because Bohdan used this method in his video.